[2022-11-0710:45:55] [ ERROR] [UserDAOH2:45] Error
org.h2.jdbc.JdbcSQLNonTransientException: IO Exception: "java.nio.file.NoSuchFileException: create.sql"; "create.sql"; SQL statement:
RUNSCRIPT FROM 'create.sql' [90031-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:554)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.message.DbException.convertIOException(DbException.java:442)
	at org.h2.command.dml.ScriptBase.openInput(ScriptBase.java:170)
	at org.h2.command.dml.RunScriptCommand.update(RunScriptCommand.java:52)
	at org.h2.command.CommandContainer.update(CommandContainer.java:169)
	at org.h2.command.Command.executeUpdate(Command.java:252)
	at org.h2.engine.Engine.openSession(Engine.java:279)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:338)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:122)
	at org.h2.Driver.connect(Driver.java:59)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:683)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:230)
	at dao.impl.UserDAOH2.save(UserDAOH2.java:33)
	at dao.impl.UserDAOH2.save(UserDAOH2.java:10)
	at service.UserService.save(UserService.java:24)
	at Main.main(Main.java:27)
Caused by: java.nio.file.NoSuchFileException: create.sql
	at java.base/sun.nio.fs.UnixException.translateToIOException(UnixException.java:92)
	at java.base/sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:106)
	at java.base/sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:111)
	at java.base/sun.nio.fs.UnixFileSystemProvider.newByteChannel(UnixFileSystemProvider.java:218)
	at java.base/java.nio.file.Files.newByteChannel(Files.java:380)
	at java.base/java.nio.file.Files.newByteChannel(Files.java:432)
	at java.base/java.nio.file.spi.FileSystemProvider.newInputStream(FileSystemProvider.java:422)
	at java.base/java.nio.file.Files.newInputStream(Files.java:160)
	at org.h2.store.fs.disk.FilePathDisk.newInputStream(FilePathDisk.java:407)
	at org.h2.store.fs.FileUtils.newInputStream(FileUtils.java:277)
	at org.h2.command.dml.ScriptBase.openInput(ScriptBase.java:168)
	... 14 more
[2022-11-0710:47:20] [ ERROR] [UserDAOH2:45] Error
org.h2.jdbc.JdbcSQLIntegrityConstraintViolationException: Unique index or primary key violation: "PRIMARY KEY ON PUBLIC.USERS(ID) ( /* key:0 */ 10, 'Andres', 'Poblete', 'user')"; SQL statement:
INSERT INTO users VALUES(?,?,?,?) [23505-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:508)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.mvstore.db.MVPrimaryIndex.add(MVPrimaryIndex.java:120)
	at org.h2.mvstore.db.MVTable.addRow(MVTable.java:519)
	at org.h2.command.dml.Insert.insertRows(Insert.java:174)
	at org.h2.command.dml.Insert.update(Insert.java:135)
	at org.h2.command.dml.DataChangeStatement.update(DataChangeStatement.java:74)
	at org.h2.command.CommandContainer.update(CommandContainer.java:169)
	at org.h2.command.Command.executeUpdate(Command.java:252)
	at org.h2.jdbc.JdbcPreparedStatement.executeUpdateInternal(JdbcPreparedStatement.java:209)
	at org.h2.jdbc.JdbcPreparedStatement.executeUpdate(JdbcPreparedStatement.java:169)
	at dao.impl.UserDAOH2.save(UserDAOH2.java:42)
	at dao.impl.UserDAOH2.save(UserDAOH2.java:10)
	at service.UserService.save(UserService.java:24)
	at Main.main(Main.java:27)
[2022-11-0710:47:20] [ ERROR] [AppointmentDAOH2:52] Error
org.h2.jdbc.JdbcSQLIntegrityConstraintViolationException: Unique index or primary key violation: "PRIMARY KEY ON PUBLIC.APPOINTMENT(ID) ( /* key:0 */ 5, 5, 5, DATE '2022-11-29', TIMESTAMP '2022-11-03 17:20:35')"; SQL statement:
INSERT INTO appointment VALUES(?,?,?,?,?) [23505-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:508)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.mvstore.db.MVPrimaryIndex.add(MVPrimaryIndex.java:120)
	at org.h2.mvstore.db.MVTable.addRow(MVTable.java:519)
	at org.h2.command.dml.Insert.insertRows(Insert.java:174)
	at org.h2.command.dml.Insert.update(Insert.java:135)
	at org.h2.command.dml.DataChangeStatement.update(DataChangeStatement.java:74)
	at org.h2.command.CommandContainer.update(CommandContainer.java:169)
	at org.h2.command.Command.executeUpdate(Command.java:252)
	at org.h2.jdbc.JdbcPreparedStatement.executeUpdateInternal(JdbcPreparedStatement.java:209)
	at org.h2.jdbc.JdbcPreparedStatement.executeUpdate(JdbcPreparedStatement.java:169)
	at dao.impl.AppointmentDAOH2.save(AppointmentDAOH2.java:48)
	at dao.impl.AppointmentDAOH2.save(AppointmentDAOH2.java:13)
	at service.AppointmentService.save(AppointmentService.java:24)
	at Main.main(Main.java:66)
